optimum/habana/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -662,7 +662,9 @@ def __call__(
                 num_samples=num_batches * batch_size
                 if t1 == t0 or use_warmup_inference_steps
                 else (num_batches - throughput_warmup_steps) * batch_size,
-                num_steps=num_batches * batch_size * num_inference_steps,
+                num_steps=num_batches * batch_size * num_inference_steps
```

Comment:
Looks good, but let's align the coding style with SDXL (pick one: either 2 use inline "if" statements or use one "if-else" statement preprocessing).  To me SDXL style looks more readable.

Alternatively, for even more consistency, we could use same approach for SD and SDXL pipelines where we measure just last batch time instead of taking average time of all batches after warmup.   
Would like to see tests with 1, 2, 3, 4 number of batches to ensure correct functionality before merging